### PR TITLE
Update power-bi-imported-query.md

### PR DESCRIPTION
**Pull request by davidwassonmsft**
Added additional download link for Power BI Desktop as the store link was not working, and the "try free" links did not work either.

### CODE REVIEW

**Comment by PRMerger13** [comment]
@davidwassonmsft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/power-bi-imported-query.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ You need the following to complete this article:
 
 * An organizational email account that is a member of Azure Active directory, so you can connect to the [Azure Data Explorer help cluster](https://dataexplorer.azure.com/clusters/help/databases/samples).
 * [Power BI Desktop](https://powerbi.microsoft.com/get-started/) (select **DOWNLOAD FREE**).
+* [Power BI Desktop - Alternate Download Link](https://www.microsoft.com/en-us/download/details.aspx?id=58494).
```
Comment (GitHubber17) [inline data-explorer/power-bi-imported-query.md:25]:
```suggestion
* [Power BI Desktop - Alternate Download Link](https://www.microsoft.com/download/details.aspx?id=58494).
```

Comment (GitHubber17) [reply data-explorer/power-bi-imported-query.md:25]: If the submission is accepted, remove the locale identifier (en-us) from the URL.

**Comment by PRMerger18** [comment]
@davidwassonmsft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by shsagir — APPROVED** [review]

**Comment by shsagir** [comment]
@davidwassonmsft Thx for the suggestion. As an MSFT, please propose future changes using the private repo. Thx
#sign-off

**Pull request merged**